README.md

```diff
@@ -1,44 +1,42 @@
 [![Build Status](https://travis-ci.org/UIUC-PPL/projections.svg?branch=master)](https://travis-ci.org/UIUC-PPL/projections)
 
-TO COMPILE PROJECTIONS:
+To Compile Projections:
```

Comment:
Can you still change the heading from underline to `#` on the same line, ie. `# To Compile Projections`?

Reply:
Ah, yeah, good point, I'll do that.